tutorials/nginx-ingress-gke/index.md

```diff
@@ -0,0 +1,350 @@
+   (Nginx Plus only), which guarantees that all the requests from the same
+   client are always passed to the same backend container.
+-  [Support for JWTs](https://github.com/nginxinc/kubernetes-ingress/blob/master/examples/jwt)
+   (Nginx Plus only), which allows Nginx Plus to authenticate requests by
+   validating JSON Web Tokens (JWTs).
+
+The following diagram shows the architecture described above:
+
+<img src="https://github.com/ameer00/community/blob/master/tutorials/nginx-ingress-gke/Nginx%20Ingress%20on%20GCP%20-%20Fig%2002.png" width="65%">
+
+This tutorial illustrates how to set up a
+[Deployment in Kubernetes](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+with an Ingress Resource using Nginx as the Ingress Controller to route/load
+balance traffic from external clients to the Deployment.  This tutorial explains how to accomplish the following:
+
+-  Create a Kubernetes Deployment
+-  Set up an Ingress Resource object for the Deployment
+-  Set up an Ingress Controller using Nginx as the controller (instead of the
+   Kubernetes Engine default Cloud Load Balancer)
+
+## Objectives
+
+-  Create a simple Kubernetes web _application_ Deployment 
+-  Create a _default backend_ Deployment to serve 404 for all non-matching
+   hosts and paths
+-  Create an _Ingress Resource_ for the _application_
+-  Configure an _Nginx Ingress_ Deployment as an Ingress Controller for the
+   _application_
+-  Test Nginx Ingress functionality by accessing the Google Cloud L4 LB
+   frontend IP and ensure it can access the web application
+
+## Costs
+
+This tutorial uses billable components of Cloud Platform, including:
+
+-  Kubernetes Engine
+-  Google Cloud Load Balancing
+
+Use the [Pricing Calculator](https://cloud.google.com/products/calculator) to
+generate a cost estimate based on your projected usage.
+
+## Before You Begin
+
+1. Create or select a GCP project.  
+   [GO TO THE PROJECTS PAGE](https://console.cloud.google.com/project)
+   
+1. Enable billing for your project.  
+	[ENABLE BILLING](https://support.google.com/cloud/answer/6293499#enable-billing)
+
+1. Enable the Kubernetes Engine API.  
+	[ENABLE APIs](https://console.cloud.google.com/flows/enableapi?apiid=container,cloudresourcemanager.googleapis.com)
+
+## Set up your environment
+
+In this section you configure the infrastructure and identities required to
+complete the tutorial.
+
+### Create a Kubernetes Engine cluster using Cloud Shell
+
+1. You can use Cloud Shell to complete this tutorial. To use Cloud Shell, perform the following steps:  
+	[OPEN A NEW CLOUD SHELL SESSION](https://console.cloud.google.com/?cloudshell=true)
+
+1. Set your project's default compute zone and create a cluster by running the following commands:  
+  
+        gcloud config set compute/zone us-central1-f  
+        gcloud container clusters create nginx-tutorial --machine-type=n1-standard-1 --num-nodes=2
```

Comment:
Its safer to do this if you are doing this tutorial on a free account.  I originally did not have this and for my setup, it was creating a n1-standard-2 machine type.  People could change their env variable and this will ensure what they're creating.

Reply:
This is incorporated